pkg/framework/coremodel/gen.go

```diff
@@ -77,6 +77,13 @@ func main() {
 		wd.Merge(wdt)
 	}
 
+	wdr, err := gcgen.GenerateCoremodelRegistry(filepath.Join(groot, "pkg", "framework", "coremodel", "registry", "registry_gen.go"), lins)
```

Comment:
you could do with using more descriptive naming here. What is `wd`? `wdr`? `wtd`?

Reply:
I get that this is generated code, but it still used and still needs to be understandable.

Reply:
This is the code generator, not generated code, but either way, fair enough.

`wd` is the overall `WriteDiffer`. i've left that one as-is. The others, `wdg`, wdt`, `wdr`, are `WriteDiffer`s produced by sub-generators, for Go, Typescript, and Registry, respectively. They all get merged into the main `wd`, which effectively batches their writes together.

i've renamed those latter three to be more suggestive.